Userland/Libraries/LibJS/Bytecode/Interpreter.cpp

```diff
@@ -340,7 +340,7 @@ void Interpreter::run_bytecode(size_t entry_point)
     auto* locals = running_execution_context.locals.data();
     auto& accumulator = this->accumulator();
     auto& executable = current_executable();
-    auto& bytecode = executable.bytecode;
+    auto const* bytecode = executable.bytecode.data();
```

Comment:
This change seems unrelated
(LibJS/Bytecode: Keep instruction source mappings in Executable)